### PR TITLE
Update ats2-postiats to 0.2.13

### PR DESCRIPTION
**Pull request by ryantking**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Also changes the description to the official one from the ats2 website

### CODE REVIEW

File: Formula/ats2-postiats.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Ats2Postiats < Formula
-  desc "ATS programming language implementation"
+  desc "ATS2: Unleashing the Potentials of Types and Templates"
```
Comment (MikeMcQuaid) [inline Formula/ats2-postiats.rb]: This description feels worth to me than before; I have no idea what the new one means.

Comment (RyanTKing) [inline Formula/ats2-postiats.rb]: It's the official one from the language designer

Comment (MikeMcQuaid) [inline Formula/ats2-postiats.rb]: I'd call that a subtitle not description. Something around `ATS is a statically typed programming language that unifies implementation with formal specification.` feels better to me.

Comment (RyanTKing) [inline Formula/ats2-postiats.rb]: Thanks for your input, I'll tweak it to something along the lines of what you suggested next time I push a version update

Comment (MikeMcQuaid) [inline Formula/ats2-postiats.rb]: Thanks @RyanTKing!

**Comment by RyanTKing** [comment]
Hope this description is more up to your standards!

**Pull request closed without merging**